odoo/tools/mail.py

```diff
@@ -36,6 +37,27 @@
      'data-publish', 'data-id', 'data-res_id', 'data-member_id', 'data-view-id'
      ])
 
+# allow base64 image
+_image_dataurl = re.compile(r'(data:image/(.+));base64,([^"\']*)', re.I)
+_is_possibly_malicious_scheme = re.compile(r'^(?:javascript|jscript|livescript|vbscript|data|about|mocha):', re.I).search
+def _is_javascript_scheme(s):
+    if _image_dataurl.search(s):
+        return None
+    return _is_possibly_malicious_scheme(s)
+lxml.html.clean._is_javascript_scheme = _is_javascript_scheme
```

Comment:
You might be able to drop this part if the substitution happens in `mail.message.create()` _before_ calling `super`, including the `import`.